src/db/models.py

```diff
@@ -1,24 +1,64 @@
 from . import connection
+import psycopg2.sql as sql
 
 class ApplicationsModel:
+    TableName = "applications"
+    Model = {}
```

Comment:
Model shouldn't be a static property

Reply:
why? it's a schema definition that should be common to all ApplicationsModel instances

Reply:
Is there a need for multiple ApplicationsModel instances, or can we do with a singleton class?

Reply:
We can save the ApplicationsModel instance and reuse it in later calls to upload_csv

Reply:
I was thinking each instance could represent a row, so something like the computation can be done using a bunch of model objects, and each instance would have it's own data, though not shown here. You'd set the data by calling get row